#uint64 timestamp
float32[3] kp
float32[3] kd
